### ISSUE TITLE
Multiple NER components in one pipeline

### ISSUE THREAD

**Issue by romlatron** [explosion/spaCy#3933]
I know this issue was already posted in #1752, but I'm actually having the exact same problem, even though it was supposed to be fixed in #2159. I am loading different NER models which are contained in the ner_models folder in the NLP pipeline, as following: 

## How to reproduce the behaviour
    for i, model in enumerate(os.listdir(path+ "/ner_models")):
        ner = EntityRecognizer(nlp.vocab)
        ner.from_disk(path+ "/ner_models/" + model, vocab=False)
        nlp.add_pipe(ner, "ner" + str(i+1))

I then get a segmentation fault when I apply the model to a sentence in which entities are detected by one of the NER (code 0xC0000005 with PyCharm), am I missing something?

## Your Environment

* **spaCy version:** 2.0.18                                                                                                                                                                                                                                                     * **Platform:** Windows-10-10.0.17134-SP0                                                                                                                                                                                                                                       * **Python version:** 3.6.6

**Comment by honnibal** [comment]
Could you try again with v2.1.4?

**Comment by romlatron** [comment]
Thanks for the answer!
We didn't upgrade to v2.1 yet, because there are some impacts on our project we cannot handle at the moment (for example retraining the model which implies upgrading Prodigy license).
Would it be possible to get this fix in v2.0?

**Comment by honnibal** [comment]
Sorry, we can't backport the fix to v2.0.18. If you diff the `spacy/syntax` module, you'll see that I did a fairly significant refactor of the parser and model code to clean it up and make it more consistent.

The release notes do link to issues we fixed though, so you might be able to find a small patch that fixes your specific bug. If you're willing to backport the small patches that affect you and can make a PR with  the relevant tests, we'd be happy to make a v2.0.19 release with the fixes.
